Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openrouteservice/openrouteservice:v8.0.0
+
+RUN wget http://download.geofabrik.de/north-america/us/massachusetts-latest.osm.pbf -O files/data.osm.pbf
+
+COPY ors-config.yml config/ors-config.yml
```

Comment:
I feel like this is less likely to change then the OSM data, can we move this layer to be before the `wget`?

Reply:
Or is it more beneficial to cache the osm data when editing the config...

Reply:
This position docker is putting us in sucks :)

Reply:
Resolving this because I don't think it matters.

Reply:
Hahaha I went through the same train of thought when I was doing this originally.

I put it in the order I did because of 👇 
> Or is it more beneficial to cache the osm data when editing the config...

In principle I agree that it's more _pure_ to have the volatile OSM data be a later step, but the only times we'd actually care about using the cache (as opposed to explicitly wanting to bust the cache) are times when the OSM data is not volatile - when we're fiddling with config or things like that.

Actually, I think the correct _pure_ approach would be to have the volatile OSM data be mounted in as a volume, which docker does allow, so I'm more inclined to blame the fact that we're not doing it that way 😂